pkg/api/v1beta4/dynakube/dynakube_webhook.go

```diff
@@ -0,0 +1,13 @@
+package dynakube
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+func SetupWebhookWithManager(mgr ctrl.Manager, validator admission.CustomValidator) error {
+	return ctrl.NewWebhookManagedBy(mgr).
+		For(&DynaKube{}).
+		WithValidator(validator). // will create an endpoint at /validate-dynatrace-com-v1beta4-dynakube
```

Comment:
pls also add it in the [validatingwebhookconfiguration.yaml](https://github.com/Dynatrace/dynatrace-operator/blob/b1e413b7440cdad7a52f9551ef22622db789091d/config/helm/chart/default/templates/Common/webhook/validatingwebhookconfiguration.yaml#L77)

Reply:
[d23217a](https://github.com/Dynatrace/dynatrace-operator/pull/4529/commits/d23217aaa208c55503711b24495c10bf7176cc39)